### DISCUSSION TITLE
What is stats Based adaptive streaming?

### DISCUSSION THREAD

**Discussion by Mohit-3196** [ant-media/Ant-Media-Server#5267] [FAQs] [answered]
In the [release logs](https://github.com/ant-media/Ant-Media-Server/releases) of AMS v2.6.0, it mentions stats-based adaptive streaming and there is a new property [`settings.statsBasedABREnabled`](https://antmedia.io/javadoc/io/antmedia/AppSettings.html#statsBasedABREnabled).

Can you please let me know how is this different from the existing adaptive bitrate mechanism in v2.5.3?

**Comment by burak-58** [comment, marked as answer]
Thanks for this great question. Let me give some basic information first.

One of the most important features of Ant Media Server (AMS) is adaptive bitrate (ABR) streaming, which means that AMS generates multiple output streams with different resolution&bitrate for an input stream. Viewers plays the most appropriate output stream according to their network condition between the viewer client machine and server. By default, AMS automatically selects the best appropriate output by the ABR algorithm.

In version v2.6.0, a new algorithm i.e. stat-based ABR is added and set as the default algorithm. I will tell details after telling the previous one.

Previously AMS used to use a bitrate estimation-based algorithm by default. AMS was making some bandwidth estimations between client and server machines and selecting the output stream with the highest bitrate less than the estimated bandwidth. Any estimation error may affect the performance.

In new stat-based algorithm, AMS monitors some WebRTC Statistics like packet lost ratio, jitter and round trip time. Then it compares their values with some thresholds and according to the comparison result it decides to switch higher or lower quality output stream.
